tcwebhooks-web-ui/src/main/resources/buildServerResources/WebHook/index.jsp

```diff
@@ -442,12 +433,13 @@
 									.removeClass('webHookRowTemplate')
 									.addClass('webHookRow')
 									.appendTo('#webHookTable > tbody');
-					jQueryWebhook("#viewRow_" + webhook.uniqueKey + " > td.webHookRowItemUrl").html(htmlEscape(webhook.url)).click(function(){BS.EditWebHookDialog.showDialog(webhook.uniqueKey, '#hookPane');});
+					jQueryWebhook("#viewRow_" + webhook.uniqueKey + " > td.webHookRowItemUrl").text(webhook.url).click(function(){BS.EditWebHookDialog.showDialog(webhook.uniqueKey, '#hookPane');});
 					if (webhook.payloadTemplate === 'none') {
-						jQueryWebhook("#viewRow_" + webhook.uniqueKey + " > td.webHookRowItemFormat").html(webhook.payloadFormatForWeb);
+						jQueryWebhook("#viewRow_" + webhook.uniqueKey + " > td.webHookRowItemFormat").text(webhook.payloadFormatForWeb);

```

Comment:
I'm pretty sure this value can contain HTML. Hence its name.

Reply:
In the other version of this code you were using your HTML escape function, so I figured this was the same situation (`editWebhook.js` line 655/650). Maybe I'm missing the difference, to be honest I didn't dig too deep into the meaning/origin of these.

Reply:
Ok. Good point

Reply:
Once you figure out the other `.*ForWeb` members one of us can change those to whichever is appropriate.

Reply:
It's safe to convert these to `text()`
1. `enabledEventsListForWeb` only contains a list of build event names comma delimited. Never user input.
1. `enabledBuildsListForWeb` also only contains a string like "All builds & Sub-Projects", or "12 Builds", and is never user input.

Reply:
You mean there's no HTML in these at all?

Reply:
Yes. Correct. I wrote those years ago before I figured I should pass a list and render the loop in the view.

It's a bit yuck as the view logic is in the app layer. Not great separation of concerns.

Reply:
Code is here: https://github.com/tcplugins/tcWebHooks/blob/1.1.x.x-js-fixes/tcwebhooks-core/src/main/java/webhook/teamcity/settings/WebHookConfig.java#L442-L458